File: 7KYU/Two numbers are positive/solution.java
public class Kata {
  
  public static boolean twoArePositive(int a, int b, int c) {
    return (a > 0 && b > 0 && c <= 0) ||
                   (a > 0 && b <= 0 && c > 0) ||
                   (a <= 0 && b > 0 && c > 0);
  }
}

File: 7KYU/Two numbers are positive/solution.php
<?php
function twoArePositive(int $a, int $b, int $c): bool {
  return ($a > 0 && $b > 0 && $c <= 0) ||
         ($a > 0 && $b <= 0 && $c > 0) ||
         ($a <= 0 && $b > 0 && $c > 0);
}